packages/api-client/src/views/Collection/CollectionAuthentication.vue

```diff
@@ -49,11 +49,15 @@ const handleToggleCollectionSecurity = () => {
 
       <RequestAuth
         class="scalar-collection-auth"
-        v-if="activeCollection?.useCollectionSecurity && activeWorkspace"
+        :class="
+          !activeCollection?.useCollectionSecurity &&
+          'pointer-events-none opacity-50 mix-blend-luminosity'
+        "
+        v-if="activeCollection && activeWorkspace"
         :collection="activeCollection"
         :envVariables="activeEnvVariables"
         :environment="activeEnvironment"
-        layout="reference"
+        layout="client"
```

Comment:
set this to client unless there's a reason not to - then we shouldn't display the section at all?